src/components/TaskForm.tsx

```diff
@@ -0,0 +1,46 @@
+// src/components/TaskForm.tsx
+import React, { useState } from 'react';
+import "../styles/TaskForm.css"
+interface TaskFormProps {
+    onSubmit: (text: string, category: string, priority: 'High' | 'Medium' | 'Low') => void;
+    initialValues?: { text: string; category: string; priority: 'High' | 'Medium' | 'Low' };
+}
+
+const TaskForm: React.FC<TaskFormProps> = ({ onSubmit, initialValues }) => {
+    const [text, setText] = useState(initialValues?.text || '');
+    const [category, setCategory] = useState(initialValues?.category || '');
+    const [priority, setPriority] = useState<'High' | 'Medium' | 'Low'>(initialValues?.priority || 'Low');
```

Comment:
Can we use enum for <'High' | 'Medium' | 'Low'> values ?

Reply:
Fixed